tests/torch/test_onnx_export.py

```diff
@@ -48,3 +51,27 @@ def test_io_nodes_naming_scheme(tmp_path):
 
         assert len(node.output) == 1
         assert node.output[0] == f"output.{idx}"
+
+
+# pylint: disable=protected-access
+@pytest.mark.parametrize('save_format, refs',
+                         (
+                             ('onnx', ('onnx', {'opset_version': PTExporter._ONNX_DEFAULT_OPSET})),
+                             ('onnx_9', ('onnx', {'opset_version': 9})),
+                             ('onnx_10', ('onnx', {'opset_version': 10})),
+                             ('onnx_11', ('onnx', {'opset_version': 11})),
+                             ('onnx_0', ValueError),
+                             ('onnx_onnx', ValueError),
+                         )
+                        )
+def test_exporter_parser_format(save_format: str, refs: Any):
```

Comment:
Might want to add another test, or a stage of the test - wherein a basic, mock model is being exported using the PTExporter and the resulting ONNX has the desired opset.